service/hub/auth/authz.go

```diff
@@ -0,0 +1,44 @@
+package auth
+
+import (
+	"context"
+	"log"
+
+	"github.com/labring/sealos/pkg/client-go/kubernetes"
+	"github.com/labring/service/hub/api"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	log.Println("authz plugin init function called")
+}
+
+type SealosAuthz struct {
+	api.Authorizer
+}
+
+func (a SealosAuthz) Authorize(ai *api.AuthRequestInfo) ([]string, error) {
```

Comment:
duplicate code！

Reply:
SealosAuthz is only one kind of authz. I think we should keep it like this.